gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/auth/JwtTokenUtil.java

```diff
@@ -36,7 +36,7 @@
 
     private static final long serialVersionUID = -2550185165626007488L;
 
-    public static final long JWT_TOKEN_VALIDITY = 5 * 60 * 60;
+    public static final long JWT_TOKEN_VALIDITY = 18000;
```

Comment:
Can this go back to 5 * 60 * 60 please?

Reply:
This was changed in the 229 PR but I am worried it wasn't merged into this one properly

Reply:
Done